src/invoices/InvoiceForm/InvoiceDocumentsForm/InvoiceDocumentFields/InvoiceDocumentFields.css

```diff
@@ -0,0 +1,17 @@
+.InvoiceDocumentField {
```

Comment:
I think we need to use lower-case first letter due to code style https://github.com/folio-org/stripes-components/blob/master/docs/CSSinStripes.md